test/httpismSpec.js

```diff
@@ -982,4 +982,29 @@ describe("httpism", function() {
       });
     });
   });
+
+  describe("json reviver", function() {
+
+    it("controls how the JSON response is deserialised", function() {
+      app.get("/", function(req, res) {
+        res.header("content-type", "application/json");
+        res.status(200).send(JSON.stringify({ blah: 1234 }));
```

Comment:
you can just `res.status(200).send({blah: 1234})` here, and skip the `Content-Type`.